src/Client/Client.fsproj

```diff
@@ -1,46 +1,43 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-    <PropertyGroup>
-        <TargetFramework>netstandard2.0</TargetFramework>
-    </PropertyGroup>
-	<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'">
-		<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-		<WarningsAsErrors />
-	</PropertyGroup>
-	<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-		<TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-		<WarningsAsErrors />
-	</PropertyGroup>
-    <ItemGroup>
-		<Compile Include="Logger.fs" />
-		<Compile Include="BoolExtensions.fs" />
-		<Compile Include="Rules.fs" />
-		<Compile Include="Pages.fs" />
-		<Compile Include="Markup.fs" />
-		<Compile Include="Styles.fs" />
-		<Compile Include="views\Menu.fs" />
-		<Compile Include="views\Rule.fs" />
-		<Compile Include="widgets\Navbar\Types.fs" />
-		<Compile Include="widgets\Navbar\State.fs" />
-		<Compile Include="widgets\Navbar\View.fs" />
-		<Compile Include="widgets\FilterBlock\Types.fs" />
-		<Compile Include="widgets\FilterBlock\State.fs" />
-		<Compile Include="widgets\FilterBlock\View.fs" />
-		<Compile Include="widgets\Pattern.fs" />
-		<Compile Include="widgets\Gender.fs" />
-		<Compile Include="widgets\Class.fs" />
-		<Compile Include="widgets\Regularity.fs" />
-		<Compile Include="widgets\Task.fs" />
-		<Compile Include="pages\NounAccusatives.fs" />
-		<Compile Include="pages\VerbParticiples.fs" />
-		<Compile Include="pages\Home.fs" />
-		<Compile Include="pages\NounPlurals.fs" />
-		<Compile Include="pages\AdjectivePlurals.fs" />
-		<Compile Include="pages\AdjectiveComparatives.fs" />
-		<Compile Include="pages\VerbImperatives.fs" />
-		<Compile Include="App.fs" />
-    </ItemGroup>
-    <ItemGroup>
-      <ProjectReference Include="..\Core\Core.fsproj" />
-    </ItemGroup>
-    <Import Project="..\..\.paket\Paket.Restore.targets" />
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'" />
+  <ItemGroup>
+    <ProjectReference Include="..\Core\Core.fsproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Logger.fs" />
+    <Compile Include="BoolExtensions.fs" />
+    <Compile Include="Rules.fs" />
+    <Compile Include="Pages.fs" />
+    <Compile Include="Markup.fs" />
+    <Compile Include="Styles.fs" />
+    <Compile Include="views/Menu.fs" />
+    <Compile Include="views/Rule.fs" />
+    <Compile Include="widgets/Navbar/Types.fs" />
+    <Compile Include="widgets/Navbar/State.fs" />
+    <Compile Include="widgets/Navbar/View.fs" />
+    <Compile Include="widgets/ImprovedInput/Types.fs" />
+    <Compile Include="widgets/ImprovedInput/State.fs" />
+    <Compile Include="widgets/ImprovedInput/View.fs" />
```

Comment:
Maybe we can rename "ImprovedInput" back to "Input" now? If this is going to be the only one to stay...

Reply:
I think it might mess up with names anyway because we'll still have "Input" in Fulma library

Reply:
Well, I guess we can live like that... Although might be a good idea to properly organize code into namespaces folder by folder (now or later), so it will be `..Widgets.Input..`